packages/mongodb-rag-core/src/contentStore/Page.ts

```diff
@@ -48,6 +48,12 @@ export type PageMetadata = {
     If the page is not versioned, this field should be undefined.
    */
   version?: VersionInfo;
+  /**
+    The page type. Other types may be added in the future. 
+    "tech-docs" indicates documents from the mongodb.com/docs site. SnootyDataSource has this type
+    "devcenter" indicates documents from the mongodb.com/developer site. DevCenterDataSource has this type
+   */
+  pageType?: "tech-docs" | "devcenter";
```

Comment:
a few things:

1. pls rename this to `contentType`. more descriptive
2. do not hard code in the values on the Page type. keep as `string`. remember, this is a flexible construct. also this type is used by pages from other sources, such as web data sources, etc.

Reply:
ah ok. in that case can you put the value inside of the Page.metadata.page object? this way its not included in the chunked data.

so the property would be `Page.metadata.page.type?: string`